src/dlgTriggerEditor.cpp

```diff
@@ -2406,69 +3274,70 @@ void dlgTriggerEditor::addTimer(bool isFolder)
 void dlgTriggerEditor::addVar(bool isFolder)
 {
     saveVar();
-    QString name;
-    mpVarsMainArea->key_type->setCurrentIndex(0);
+    mpVarsMainArea->comboBox_variable_key_type->setCurrentIndex(0);
     if (isFolder) {
         // Edbee doesn't have a readonly option, so I'm using setEnabled
         mpSourceEditorEdbee->setEnabled(false);
-        mpVarsMainArea->var_type->setDisabled(true);
-        mpVarsMainArea->var_type->setCurrentIndex(4);
-        mpVarsMainArea->lineEdit_var_name->setText("");
-        mpVarsMainArea->lineEdit_var_name->setPlaceholderText("Table name...");
+        mpVarsMainArea->comboBox_variable_value_type->setDisabled(true);
+        mpVarsMainArea->comboBox_variable_value_type->setCurrentIndex(4);
+        mpVarsMainArea->lineEdit_var_name->setText(QString());
+        mpVarsMainArea->lineEdit_var_name->setPlaceholderText(tr("Table name..."));
 
         clearDocument(mpSourceEditorEdbee, QLatin1Literal("NewTable"));
-        name = "";
     } else {
         // Edbee doesn't have a readonly option, so I'm using setEnabled
         mpSourceEditorEdbee->setEnabled(true);
-        mpVarsMainArea->lineEdit_var_name->setText("");
```

Comment:
Tut, tut - An empty raw string literal - not the right thing at all for something expecting a `QString` methinks...